src/Main.elm

```diff
@@ -632,17 +671,130 @@ modalView : Model -> Html Msg
 modalView model =
     div maskStyle
         [ modalContent []
-            [ windowsChrome [ style "padding" "0 18px 18px" ]
+            [ windowsChrome [ style "padding" "0 18px 90px" ]
                 [ formGroup "Difficulty"
                     [ radiobutton "Beginner" Beginner model.game
                     , radiobutton "Intermediate" Intermediate model.game
                     , radiobutton "Expert" Expert model.game
+                    , customRadioButton model.menu
                     ]
                 ]
             ]
         ]
 
 
+testDisplay : Maybe Menu -> Html Msg
+testDisplay menu =
+    label [ style "display" "flex", style "align-items" "center" ]
+        [ input
+            [ type_ "radio"
+            , name "value"
+            , onClick (OpenMenu (CustomDifficultyMenu 50 50 50))
+            , checked True
+            , style "margin" "4px 8px"
+            ]
+            []
+        , text "Custom Difficulty"
+        ]
+
+
+radiobutton : String -> Difficulty -> Difficulty -> Html Msg
+radiobutton settingLabel difficulty currentGameDifficulty =
+    label [ style "display" "flex", style "align-items" "center" ]
+        [ input
+            [ type_ "radio"
+            , name "value"
+            , onClick (SetDifficulty difficulty)
+            , checked (difficulty == currentGameDifficulty)
+            , style "margin" "4px 8px"
+            ]
+            []
+        , text settingLabel
+        ]
+
+
+customRadioButton : Maybe Menu -> Html Msg
```

Comment:
this function should take a `Menu` instead of a `Maybe Menu`. in general with `Maybe`, it tends to be better to leave the uncertainty outside of the function and use `Maybe.map` to say "apply this function to my uncertain value".

currently, there's a 

```elm
Nothing ->
  div [] []
```

below that will never get hit. if we're rendering this `customRadioButton`, then we have a `Menu`! if you ever notice that you're handling an impossible `case`, then look to refactor.